kie-wb-common-react/import-java-classes/src/components/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
is this expected to be empty file?

Reply:
@vpellegrino Yes, In my local I have this file populated and components are done as well. The scope of this PR is to add the project structure only, just to be sure is correct. In future PR I'll add the components and populate this file as well.